### PR TITLE
Note addition to the Countermeasure section

### LINKED ISSUE #9523: Please add a note
Please add this note in Countermeasure part : if the a service account is configured in log on properties of a windows service it requires the network log on rights to the domain controllers, to start properly. 

Daniele Bona
Security & Identity

dabona@microsoft.com

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As requested in issue ticket #9523 (**Please add a note**), the aim of this PR is to add a note to the Countermeasure section of the document article "Deny access to this computer from the network".

Thanks to Daniele Bona (dbona75) (dabona@microsoft.com) for the request.

**Proposed change:**

- Add a Note blob explaining the required Network Logon rights to the domain controllers.

**Codestyle & whitespace changes:**

- Remove any redundant end-of-line (EOL) blanks.
- Standardize indent whitespace in numbered lists and bullet point lists to one single blank space.

**Additional notes:**

- The added note could possibly benefit from improved phrasing, I am not sure if the current phrasing is clear enough.
- I also updated the article date to 05/19/2021 (from 04/19/2017), but this may be changed to a more accurate date by the person merging this PR.

**Ticket closure or reference:**

Closes #9523

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
Regular source file view:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9531/files

To skip the lines containing only whitespace reduction, Rich Diff view:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9531/files?diff=unified&w=1

**Comment by illfated** [comment]
#sign-off

File: windows/security/threat-protection/security-policy-settings/deny-access-to-this-computer-from-the-network.md [hunk 1]
```diff
@@ -93,13 +93,17 @@ Users who can log on to the device over the network can enumerate lists of accou
...
+- Anonymous logon
+- Built-in local Administrator account
+- Local Guest account
+- All service accounts
 
 An important exception to this list is any service accounts that are used to start services that must connect to the device over the network. For example, let’s say you have configured a shared folder for web servers to access, and you present content within that folder through a website. You may need to allow the account that runs IIS to log on to the server with the shared folder from the network. This user right is particularly effective when you must configure servers and workstations on which sensitive information is handled because of regulatory compliance concerns.
 
+> [!NOTE]
+> If the service account is configured in the logon properties of a Windows Service,
+> it requires Network Logon rights to the domain controllers to start properly.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/deny-access-to-this-computer-from-the-network.md:105]:
```suggestion
> If the service account is configured in the logon properties of a Windows service, it requires network logon rights to the domain controllers to start properly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**